docs/reference/management_commands.md

```diff
@@ -97,6 +97,15 @@ You can prevent the `update_index` command from indexing any data by using the `
 $ python manage.py update_index --schema-only
 ```
 
+### Silencing the command
+You can prevent logs to the console by providing `--verbosity 0` as an arguement:
+
+```console
+$ python manage.py update_index --verbosity 0
+```
+
+Any number > 1 will produce the same logs.
```

Comment:
```suggestion
If this is omitted or provided with any number above 0 it will produce the same logs.
```

Just a minor re-word, will commit this on merging.